DatadogRUM/Sources/Instrumentation/AppHangs/FatalAppHangsHandler.swift

```diff
@@ -92,6 +93,7 @@ internal final class FatalAppHangsHandler {
 
             let realErrorDate = fatalHang.hang.startDate.addingTimeInterval(fatalHang.serverTimeOffset)
             let realDateNow = dateProvider.now.addingTimeInterval(context.serverTimeOffset)
+            let timeSinceAppStart = fatalHang.appLaunchDate.map { realErrorDate.timeIntervalSince($0) }
```

Comment:
FYI, there was a bug in unit conversion (multiplying by `1_000` while the value is `TimeInterval`, so must be in seconds). I pushed a fix with adding tests to `timeSinceAppStart` for Fatal App Hangs. It is all good now.